src/runtime_src/core/include/xrt/xrt_kernel.h

```diff
@@ -701,6 +701,9 @@ class kernel
   read_register(uint32_t offset) const;
 
 public:
+  std::string get_name() const;
+  xrt::xclbin get_xclbin() const;
```

Comment:
Yes, please document doxygen style since these are public APIs.  Also please follow the suggested convention:
```suggestion
  XCL_DRIVER_DLLESPEC
  std::string 
  get_name() const;
  
  XCL_DRIVER_DLLESPEC
  xrt::xclbin 
  get_xclbin() const;
```
I am adamant about coding to the left; I simply hate scanning code for the function name somewhere after the return type which varies in length always.

Reply:
ok to test